packages/docusaurus/src/server/__tests__/configValidation.test.ts

```diff
@@ -111,25 +110,20 @@ describe('normalizeConfig', () => {
   });
 
   it('normalizes various URLs', () => {
-    const consoleMock = jest
-      .spyOn(console, 'warn')
-      .mockImplementation(() => {});
-
     expect(
       normalizeConfig({
         url: 'https://mysite.com/',
       }).url,
     ).toBe('https://mysite.com');
-    expect(
+    expect(() =>
       normalizeConfig({
         // This shouldn't happen
         url: 'https://mysite.com/foo/',
-      }).url,
-    ).toBe('https://mysite.com/foo');
-
-    expect(consoleMock.mock.calls[0][0]).toMatchInlineSnapshot(
-      `"[WARNING] Docusaurus config validation warning. Field "url": The url is not supposed to contain a sub-path like '/foo/'. Please use the baseUrl field for sub-paths."`,
-    );
+      }),
+    ).toThrowErrorMatchingInlineSnapshot(`
+      "The url is not supposed to contain a sub-path like "". Please use the baseUrl field for sub-paths.
+      "
+    `);
```

Comment:
I'm not sure we need this anymore, at least not in this test group, because here it's not testing URL normalization but URL rejection so this feels weird